spaces/domains.json

```
@@ -119,6 +119,7 @@
   "signal.goldmining.farm": "gmt",
   "gov.aragon.org": "aragon",
   "vote.yogi.fi": "yogi-fi.eth",
+  "vote.yakuzacats.com":"ycsdao.eth",
```

Please add a space between your sub domain and your ENS.
```suggestion
  "vote.yakuzacats.com": "ycsdao.eth",
```